hoot-core/src/main/cpp/hoot/core/io/HootApiDb.cpp

```diff
@@ -297,6 +297,14 @@ void HootApiDb::deleteMap(long mapId)
   // Drop related renderDB First
   dropDatabase(_getRenderDBName(mapId));
 
+  // Drop related sequences
+  dropSequence(getCurrentRelationMembersSequenceName(mapId));
+  dropSequence(getCurrentRelationsSequenceName(mapId));
+  dropSequence(getCurrentWayNodesSequenceName(mapId));
+  dropSequence(getCurrentWaysSequenceName(mapId));
+  dropSequence(getCurrentNodesSequenceName(mapId));
+  dropSequence(getChangesetsSequenceName(mapId));
+
```

Comment:
Created `dropSequence` method to consolidate the SQL into one place.  Removing sequences before removing the tables.